provision/vvv-hosts

```diff
@@ -1 +1 @@
-yoast-acf-analysis.vvv.dev
\ No newline at end of file
+acf-content-analysis-for-yoast-seo.test
```

Comment:
this file isn't necessary if you've already got the hosts in `vvv-custom.yml`, support for `vvv-hosts` is mostly for convenience and backwards compatibility with VVV 1